Issue838 t sup air multi zone

JayHuLBL opened the pull request:
The sequence defines supply air temperature control for multilane AHU, according to G36, Part5.N.2

mwetter commented on Buildings/Controls/OBC/ASHRAE/G36/Atomic/SupplyAirTempSet_MultiZone.mo:
```diff
@@ -0,0 +1,330 @@
+        iconTransformation(extent={{100,-10},{120,10}})));
+
+
+//   fixme: Need a Trim&Response logic to find the T_max
```
Address or remove the fixme

mwetter commented on Buildings/Controls/OBC/ASHRAE/G36/Atomic/SupplyAirTempSet_MultiZone.mo:
```diff
@@ -0,0 +1,330 @@
+  CDL.Logical.Switch swi3 "Check output regarding supply fan status"
+    annotation (Placement(transformation(extent={{100,20},{120,0}})));
+  CDL.Integers.LessThreshold intLesThr(threshold=Constants.OperationModes.warUpInd)
+    "Check if operation mode index is less than warm-up mode index (4)"
```
It is better to not state the numerical value. Otherwise, if we ever need to change `Constants.OperationModes`, the comment will be wrong, and we probably won't detect it

mwetter commented on Buildings/Controls/OBC/ASHRAE/G36/Atomic/SupplyAirTempSet_MultiZone.mo:
```diff
@@ -0,0 +1,330 @@
+within Buildings.Controls.OBC.ASHRAE.G36.Atomic;
+block SupplyAirTempSet_MultiZone
```
As this is only for VAV systems, the block should have `VAV` in its name, or somewhere in its package name.

milicag replied:
@JayHuLBL could this block have a name equivalent to the single zone one (without the fan speed)?

JayHuLBL commented:
@mwetter Your comments have been addressed:
- [x] renamed the block
- [x] added TrimRespond logic (the same one used in #866) and implemented in the `VAVMultiZoneSupTempSet` block

mwetter commented:
@JayHuLBL : Please make the following changes, then it is ready to merge:
- [ ] Rename `VAVMultiZoneSupTempSet` to `VAVMultiZoneTSupSet`, as there is also a block `VAVSingleZoneTSupSet`. Then, they have the same name, except for 'Single' and 'Multi'.
- [ ] Sometimes we use TrimRespond while the documentation uses "trim-response". Please use consistently either "respond" or "response", whatever term Guideline 36 uses.
